start_redact.sh

```diff
@@ -12,17 +12,21 @@ else
     installation_dir="$INSTALLATION_DIR"
 fi
 source "$installation_dir/docker-compose.env"
-set +a
 
-while getopts "ua" opt; do
+while getopts "uah" opt; do
     case $opt in
         u) ui="SET"
         ;;
         a) attach="SET"
         ;;
+        h) source "$installation_dir/high-throughput.env"
+        ;;
     esac
 done
 
+# turn of env var marking from sourced env files
```

Comment:
off?

Reply:
Did you test this? +a should turn off making the defined variables available to subcommands. 
'env var marking' is not really directly understandable I also believe?

Reply:
I did test it, and it only worked (env var values being passed to docker compose commands) with `set +a` being run after the importing of both env files. Although I don't fully understand the concept of env var marking...

Reply:
I am adding a clarification.